data/json/items/comestibles/drink.json

```diff
@@ -1008,6 +1008,18 @@
     "flags": [ "EATEN_HOT" ],
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_willowbark",
+    "copy-from": "tea",
+    "name": { "str_sp": "willowbark tea" },
+    "healthy": -1,
+    "fatigue_mod": 0,
+    "stim": 0,
+    "description": "A bitter tea made by boiling willow bark in water, used since ancient times as a mild painkiller.",
+    "fun": -2,
```

Comment:
maybe remove the `EATEN_HOT` flag inherited from tea?

Reply:
``
```suggestion
    "fun": -2,
    "delete": { "flags": [ "EATEN_HOT" ] },
```